warehouse/identities.go

```diff
@@ -442,24 +440,13 @@ func (wh *HandleT) populateHistoricIdentities(warehouse model.Warehouse) {
 		}
 		defer whManager.Cleanup()
 
-		schemaHandle := SchemaHandle{
-			warehouse:    job.warehouse,
-			stagingFiles: job.stagingFiles,
-			dbHandle:     job.dbHandle,
-			whSchemaRepo: repo.NewWHSchemas(job.dbHandle),
-		}
-		job.schemaHandle = &schemaHandle
-
-		job.schemaHandle.schemaInWarehouse, job.schemaHandle.unrecognizedSchemaInWarehouse, err = whManager.FetchSchema()
+		err = job.schemaHandle.fetchSchemaFromWarehouse(whManager)
 		if err != nil {
 			pkgLogger.Errorf(`[WH]: Failed fetching schema from warehouse: %v`, err)
 			job.setUploadError(err, model.Aborted)
 			return
 		}
 
-		schemaHandle.SkipDeprecatedColumns(job.schemaHandle.schemaInWarehouse)
-		schemaHandle.SkipDeprecatedColumns(job.schemaHandle.unrecognizedSchemaInWarehouse)
```

Comment:
How come we removed these? What were they doing exactly? Can you explain this a bit?

Reply:
We are calling this internally instead of exposing it.
https://github.com/rudderlabs/rudder-server/blob/d1f967f05d6f738bf93fdcf1df3f8bafacd5f06d/warehouse/schema.go#L130